lib/browser.ts

```diff
@@ -679,8 +679,12 @@ export class ProtractorBrowser extends Webdriver {
                     let $timeout = $delegate;
 
                     let taskId = 0;
-                    if (!window['NG_PENDING_TIMEOUTS']) {
-                      window['NG_PENDING_TIMEOUTS'] = {};
+
```

Comment:
Thanks for fixing here - we should actually just move this whole function into `clientSideScripts` so we don't deal with TS here at all. 
I filed an issue at https://github.com/angular/protractor/issues/3550